### PR TITLE
Reword excessively hardcore requirement

### PR DESCRIPTION
**Pull request by RobinHood70**
While hilarious as is, this should probably be reworded.

### CODE REVIEW

**Comment by cmb69** [comment]
> While hilarious as is, this should probably be reworded.

Yeah, but while we're at it, we should remove the personalization ("your") right away, at least for that paragraph.

**Comment by RobinHood70** [comment]
> Yeah, but while we're at it, we should remove the personalization ("your") right away, at least for that paragraph.

Done for the whole file. How does it look?

File: reference/errorfunc/functions/set-error-handler.xml [hunk 1]
```diff
@@ -18,26 +18,26 @@
    errors in a script.
   </para>
   <para>
-   This function can be used for defining your own way of handling errors
-   during runtime, for example in applications in which you need to do
-   cleanup of data/files when a critical error happens, or when you need
-   to trigger an error under certain conditions (using
-   <function>trigger_error</function>).
+   This function can be used to define custom error handlers during runtime,
+   for example in applications which need to file/data cleanup when a critical
```
Comment (cmb69) [inline reference/errorfunc/functions/set-error-handler.xml:22]:
```suggestion
   for example in applications which need to do file/data cleanup when a critical
```
or something like that, I think.

**Review by cmb69 — COMMENTED** [review]
Pretty good. Thank you!

**Comment by cmb69** [comment]
Oh, there has been trailing whitespace which made the whitespace-checker scream. I've removed that, squashed and force-pushed (otherwise the whitespace-checker would still complain).

Thank you for the PR!

**Pull request merged**